extensions/wikia/EmbeddableDiscussions/tests/AttributesValidatorTest.php

```diff
@@ -0,0 +1,57 @@
+<?php
+
+class AttributesValidatorTest extends WikiaBaseTest {
+
+	protected function setUp() {
+		$this->setupFile = dirname( __FILE__ ) . '/../AttributesValidator.class.php';
+		parent::setUp();
+	}
+
+	/**
+	 * @dataProvider boolishDataProvider
+	 */
+	public function testIsBoolish( $value, $expected, $message ) {
+		$result = AttributesValidator::isBoolish( $value );
+		$this->assertEquals( $expected, $result, $message );
+	}
+
+	public function boolishDataProvider() {
+		return [
+			// value, expected, message
+			[ null, false, "null should be false" ],
```

Comment:
I propose to change 'should be false' 'should be true' to 'does not validate as a boolean' 'does validate as a boolean'